headers/validation.h

```diff
@@ -1264,16 +1264,10 @@ class Validator {
 			WA(name, ": Want integer, found nothing");
 		}
 		long long v;
-		try {
-			auto begin = s.c_str(), end = begin + s.size();
-			auto [ptr, ec] = std::from_chars(begin, end, v);
-			if(ptr != end or ec != std::errc{})
-				WA(name, ": Parsing " + s + " as long long failed! Did not process all characters");
-		} catch(const std::out_of_range& e) {
-			WA(name, ": Number " + s + " does not fit in a long long!");
-		} catch(const std::invalid_argument& e) {
-			WA("Parsing " + s + " as long long failed!");
-		}
+		auto begin = s.c_str(), end = begin + s.size();
+		auto [ptr, ec] = std::from_chars(begin, end, v);
+		if(ptr != end or ec != std::errc{})
```

Comment:
can you explicitly add an out-of-range check, like here:
https://en.cppreference.com/w/cpp/utility/from_chars